### PR TITLE
Improve xyz reader

### LINKED ISSUE #342: Improvement of xyz reader
Hi

I would like to suggest a very small improvement of the xyz reader. The regex is already flexible but it does not consider possible whitespaces at the beginning of the line.

**Describe the "bug"**
If you try to read an xyz file where there is a white space at the beginning of the line the returned `atoms` list of the `dasg_bio.utils.xyz_reader.read_xyz()` function is empty.

For example, you can try using an xyz file from : http://www.nanotube.msu.edu/fullerene/fullerene.php?C=34

**Correction**
The correction (if it may be call like this) is simply to add a possible whitespace at the beginning of the regex. Line 50-52 of `xyz_reader.py`. The current regex is:
```python
r = re.search(
    r'^([\w]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)', 
    line)
```
The following regex accept white spaces at the beginning of the line:

```py
r'^\s*([\w]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)'
```

The other solution is to remove the `^` character but it might me less safe.

Thank you for this nice dash module

### PR DESCRIPTION
**Pull request by shammamah**
Closes #342 

## About
- [ ] This is a new component
- [ ] I am adding a feature to an existing component, or improving an existing feature
- [x] I am closing an issue

## Description of changes
* Add support for whitespaces at beginning and end of line for XYZ file parser. 

## Before merging
- [x] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [x] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [x] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [0.0.11]
+
+### Added
+* Added to xyz_reader the ability to handle whitespace at the
+beginning and end of a line.
```
Comment (mkcor) [inline CHANGELOG.md]:
```suggestion
  beginning and end of a line (in data file).
```

File: dash_bio/utils/xyz_reader.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def read_xyz(
             # an example line in a xyz file:
             # C    +0.125    -1.032    +2.000
             r = re.search(
-                r'^([\w]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)',
+                r'^\s*([\w]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)\s*',
```
Comment (mkcor) [inline dash_bio/utils/xyz_reader.py]: I get the `\s` to match a whitespace character, but why the `*`, just to clarify? Sorry, I'm not a regex expert and I know conventions change across languages...

Comment (shammamah) [reply dash_bio/utils/xyz_reader.py]: `*` means 0 or more!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [0.0.11]
+
+### Added
+* Added to xyz_reader the ability to handle whitespace at the
```
Comment (mkcor) [inline CHANGELOG.md]:
```suggestion
* Added to xyz_reader the ability to handle whitespaces at the
```

File: dash_bio/utils/xyz_reader.py (hunk 2 above)
Comment (mkcor) [reply dash_bio/utils/xyz_reader.py]: Oh, duh, thanks! Then suggesting github.com/plotly/dash-bio/pull/343/files#r280881535.

File: CHANGELOG.md (hunk 3 above)
Comment (shammamah) [reply CHANGELOG.md]: I think this is fine as it is. Whitespace can refer to one or more spaces. 

https://en.wikipedia.org/wiki/Whitespace_character

>In computer programming, whitespace is any character or **series of characters** that represent horizontal or vertical space in typography.

**Pull request merged**